pgbouncer-mixin/variables.libsonnet

```diff
@@ -0,0 +1,75 @@
+local g = import './g.libsonnet';
+local var = g.dashboard.variable;
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+
+// Generates chained variables to use on on all dashboards
+{
+  new(this, varMetric):
```

Comment:
Do you need a cluster selector defined for when k8s is enabled?

Reply:
@JonathanWamsley I think this is a great question. I am ok with the k8s "cluster" related stuff being added as the "final" generic "mixin" related story in vivify (which will contain any sort of cleanup/finalization tasks) after all the individual dashboard and alert pieces are in.